generators/app/index.js

```diff
@@ -95,313 +91,6 @@ module.exports = class extends Generator {
       })
   }
 
-  spa () {
```

Comment:
pretty sure you deleted way more code here than you should have

we very much intend to continue supporting the SPA feature

you should test it to ensure it still works